tests/dialects/test_duckdb.py

```diff
@@ -754,12 +754,56 @@ def test_duckdb(self):
         )
 
         with self.assertRaises(UnsupportedError):
+            # bq has the position arg, but duckdb doesn't
             transpile(
                 "SELECT REGEXP_EXTRACT(a, 'pattern', 1) from table",
                 read="bigquery",
                 write="duckdb",
                 unsupported_level=ErrorLevel.IMMEDIATE,
             )
+        with self.assertRaises(UnsupportedError):
+            # duckdb has the group arg, but bq doesn't
+            transpile(
+                "SELECT REGEXP_EXTRACT(a, 'pattern', 2) from table",
+                read="duckdb",
+                write="bigquery",
+                unsupported_level=ErrorLevel.IMMEDIATE,
+            )
+        self.validate_all(
+            "SELECT REGEXP_EXTRACT(a, 'pattern') FROM t",
+            read={
+                "duckdb": "SELECT REGEXP_EXTRACT(a, 'pattern') FROM t",
+                "bigquery": "SELECT REGEXP_EXTRACT(a, 'pattern') FROM t",
+                "snowflake": "SELECT REGEXP_SUBSTR(a, 'pattern') FROM t",
+            },
+            write={
+                "duckdb": "SELECT REGEXP_EXTRACT(a, 'pattern') FROM t",
+                "bigquery": "SELECT REGEXP_EXTRACT(a, 'pattern') FROM t",
+                "snowflake": "SELECT REGEXP_SUBSTR(a, 'pattern') FROM t",
+            },
+        )
+        self.validate_all(
+            "SELECT REGEXP_EXTRACT(a, 'pattern', 2) FROM t",
+            read={
+                "duckdb": "SELECT REGEXP_EXTRACT(a, 'pattern', 2) FROM t",
+                "snowflake": "SELECT REGEXP_SUBSTR(a, 'pattern', 1, 1, '', 2) FROM t",
+            },
+            write={
+                "duckdb": "SELECT REGEXP_EXTRACT(a, 'pattern', 2) FROM t",
+                "snowflake": "SELECT REGEXP_SUBSTR(a, 'pattern', 1, 1, 'c', 2) FROM t",
+            },
+        )
```

Comment:
It's superfluous to add `duckdb` twice here, as in essence that tests `DuckDB (read) -> DuckDB (query) -> DuckDB (write)`. Let's add it once in either read or write dict for all these tests.

Sidenote: Are these cases tested on both DuckDB and Snowflake?

Reply:
Will do. Will check snowflake. I'm assuming we only want them in one place.